dev/tests/acceptance/tests/functional/Magento/FunctionalTest/Store/ActionGroup/AdminCreateStoreGroupActionGroup.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminCreateStoreGroupActionGroup">
+        <arguments>
+            <argument name="Website" defaultValue="_defaultWebsite"/>
+        </arguments>
+        <amOnPage url="{{AdminSystemStoreGroupPage.url}}" stepKey="navigateToNewStoreGroup"/>
+        <waitForPageLoad stepKey="waitForPageLoad1" />
+
+        <!--Create Store Group-->
```

Comment:
This comment is redundant from on my opinion. Because name of action group tells me what happens here.
I suggest to add action:
`<comment userInput="Creating Store Group" stepKey="storeGroupCreation" />`
It will make comment in console while test running.